src/models/training/t5_trainer.py

```diff
@@ -0,0 +1,131 @@
+import json
```

Comment:
I don't like this "training" folder. Clearly here not only training is happening, but also inference.
Let's keep it simple until we understand more of the code.
Also, if we are in the "training" folder, we should specify "training" for each and every file